src/tests/bsd/Mybuild

```diff
@@ -0,0 +1,8 @@
+package embox.test.BSD_queue
```

Comment:
Please, rename package to `embox.test.bsd`, as we generally use lower case names for packages.

Reply:
yes. I will do it right now.